test/FT-voting-quadratic/RIFGovernorFtQuadratic.test.js

```diff
@@ -171,21 +183,64 @@ describe('Governance - Fungible tokens voting', () => {
             reason,
           );
       });
+    });
 
+    describe('Voting results', () => {
       it('total votes should equal voters rif amount square root sum', async () => {
         const againstVotes = await governor.getAgainstVotes(proposalId);
-        expect(againstVotes).to.equal(
-          Math.sqrt(voters[0].rifAmount) + Math.sqrt(voters[1].rifAmount),
-        );
+        expect(againstVotes).to.equal(Math.sqrt(voters[1].rifAmount));
         const forVotes = await governor.getForVotes(proposalId);
-        expect(forVotes).to.equal(Math.sqrt(voters[2].rifAmount));
-        console.log(forVotes, againstVotes);
+        expect(forVotes).to.equal(
+          Math.sqrt(voters[2].rifAmount) + Math.sqrt(voters[0].rifAmount),
+        );
+      });
+
+      it('should calculate the quorum correctly', async () => {
+        const quorum = await governor.quorum(
+          (await hre.ethers.provider.getBlockNumber()) - 1,
+        );
+        expect(quorum).to.equal(Math.floor(Math.sqrt(totalSupply)));
+      });
+
+      it('quorum should be reached', async () => {
+        expect(await governor.quorumReached(proposalId)).to.be.true;
       });
 
-      // TODO: Test fails. Figure out why
       it('voting should be successfull', async () => {
         expect(await governor.voteSucceeded(proposalId)).to.be.true;
       });
     });
+
```

Comment:
So above we have a "happy path" test where the vote was successful.
Also need another one (different proposal) where the outcome is that vote was unsuccessful.